### ISSUE TITLE
Meta fields lost when other extensions are saved

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/edd-compare-products#36]
See https://github.com/easydigitaldownloads/edd-social-discounts/issues/21#issuecomment-401085634

**Comment by dgoldak** [comment]
@pippinsplugins Reopening this issue, partly fixed but issue still exists. Compare Products is still resetting Social Discounts config. 

1. Uninstalled v1.1.2 Compare Products plugin.
2. Installed v1.1.3 Compare Products plugin.
3. Set new Compare Products table configuration
4. Checked Social Discount configuration -- configuration overwritten
5. Restored Social Discounts values
6. Checked Compare Products table configuration -- not overwritten

Bottom Line:
Updating/saving Compare Products table clears Social Discounts configuration
Updating/saving Social Discounts configuration does not affect Compare Products table configuration
